src/main/fc/fc_core.c

```diff
@@ -245,6 +246,13 @@ void updateArmingStatus(void)
             }
         }
 
+        if (IS_RC_MODE_ACTIVE(BOXGRAVEYARD) && graveyardModeAllowed) {
+            setArmingDisabled(ARMING_DISABLED_GRAVEYARD);
+            preventModeChanges(); // TODO: Delay 100ms
```

Comment:
Is there a pattern in the codebase that makes delaying code easy?

Reply:
Have a look at https://github.com/betaflight/betaflight/blob/master/src/main/fc/fc_dispatch.h